docs/data/migration/migration-tree-view-v7/migration-tree-view-v7.md

```diff
@@ -226,6 +226,46 @@ All the new Tree Item-related components and utils (introduced in the previous m
 + } from '@mui/x-tree-view/TreeItemLabelInput';
 ```
 
+## Stop using `publicAPI` methods in the render
+
+The Tree Items are now memoized to improve the performances of the Tree View components.
+If you call a `publicAPI` method in the render of an item, it might not re-render and you might not have the new value.
+
+```ts
+function CustomTreeItem(props) {
+  const { publicAPI } = useTreeItemUtils();
```

Comment:
This looks like a typo, shouldn't it be 

```suggestion
  const { publicApi } = useTreeItemUtils();
```

per the rest of the API conversion? We could add this to https://mui.com/material-ui/guides/api/.

Reply:
Both remark are totally valid.
It's `publicAPI` ( / `publicApi`) internally, but when exposed it should probably be `api`